flask_restx/api.py

```
@@ -651,12 +651,9 @@ def handle_error(self, e):
 
         # When propagate_exceptions is set, do not return the exception to the
         # client if a handler is configured for the exception.
-        if (
-            not isinstance(e, HTTPException)
-            and current_app.propagate_exceptions
```

Why is propagate exceptions being removed here

---

I want all exeptions if not handled by restx return to flask, therefore some extensions like flask-jwt-extended can work properly. The purpose of `propagate_exceptions` is [for testing and debugging](https://flask.palletsprojects.com/en/1.1.x/config/#PROPAGATE_EXCEPTIONS).

I see that restx's design is make the whole application for api only - by catch all exceptions and response to json, but what if i want the route /api to only handle api and the rest serves html?